monitoring/v3/src/test/java/com/example/AlertIT.java

```diff
@@ -94,24 +95,42 @@ public void testReplaceChannels() throws IOException {
   }
 
   @Test
-  public void testDisableEnablePolicies() throws IOException {
+  public void testDisableEnablePolicies() throws IOException, InterruptedException {
     AlertSample.main(new String[] {"enable", "-d", "display_name='test-policy'"});
-    
+
     // check the current state of policy to make sure
     // not to enable the policy that is already enabled.
     boolean isEnabled = bout.toString().contains("already");
-    if (isEnabled) {
-      AlertSample.main(new String[] {"disable", "-d", "display_name='test-policy'"});
-      assertTrue(bout.toString().contains("disabled"));
+    int maxAttempts = 10;
+    int attempt = 0;
+    int factor = 1;
+    Boolean retry = true;
+    while (retry) {
+      try {
+        if (isEnabled) {
+          AlertSample.main(new String[] {"disable", "-d", "display_name='test-policy'"});
+          assertTrue(bout.toString().contains("disabled"));
 
-      AlertSample.main(new String[] {"enable", "-d", "display_name='test-policy'"});
-      assertTrue(bout.toString().contains("enabled"));
-    } else {
-      AlertSample.main(new String[] {"enable", "-d", "display_name='test-policy'"});
-      assertTrue(bout.toString().contains("enabled"));
+          AlertSample.main(new String[] {"enable", "-d", "display_name='test-policy'"});
+          assertTrue(bout.toString().contains("enabled"));
+        } else {
+          AlertSample.main(new String[] {"enable", "-d", "display_name='test-policy'"});
+          assertTrue(bout.toString().contains("enabled"));
 
-      AlertSample.main(new String[] {"disable", "-d", "display_name='test-policy'"});
-      assertTrue(bout.toString().contains("disabled"));
+          AlertSample.main(new String[] {"disable", "-d", "display_name='test-policy'"});
+          assertTrue(bout.toString().contains("disabled"));
+        }
+        retry = false;
+      } catch (StatusRuntimeException e) {
+        System.out.println("Error: " + e.toString());
+        System.out.println("Retrying...");
+        Thread.sleep(2300 * factor);
```

Comment:
Start at 1300 * a random value * factor <-- so that multiple tests running at the same time don't do this at the same time. (ie at least one will break free / round). 
https://cloud.google.com/storage/docs/exponential-backoff for more info